doc/changes.rst

```diff
@@ -1,6 +1,113 @@
 Changelog
 =========
 
+Release v0.3.0 (2021/02/15)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4522136.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4522136
+
+Highlights
+
+* 🎉 **Third minor release of PyGMT** 🎉
+* Wrap inset (`#788 <https://github.com/GenericMappingTools/pygmt/pull/788>`__) for making overview maps and subplot (`#822 <https://github.com/GenericMappingTools/pygmt/pull/822>`__) for multi-panel figures
+* Apply standardized formatting conventions (`#775 <https://github.com/GenericMappingTools/pygmt/pull/775>`__) across most documentation pages
+* Drop Python 3.6 support (`#699 <https://github.com/GenericMappingTools/pygmt/pull/699>`__) so PyGMT now requires Python 3.7 or newer
+
+New Features
+
+* Wrap grd2cpt (`#803 <https://github.com/GenericMappingTools/pygmt/pull/803>`__)
+* Let Figure.text() support record-by-record transparency (`#716 <https://github.com/GenericMappingTools/pygmt/pull/716>`__)
+* Provide basic support for FreeBSD (#700, #878)
```

Comment:
Seems like the `sed` command missed a few, looks like we forgot to update it after https://github.com/GenericMappingTools/pygmt/pull/683#discussion_r523388058. Try
```
sed -i.bak -E 's$\(#([0-9]*)\)$(`#\1 <https://github.com/GenericMappingTools/pygmt/pull/\1>`__)$g' changes.rst
```

Reply:
Manually switched the formatting for some of the PR refs; the problem was when they were both in a set of parentheses.